chains/alephzero/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+blockExplorers:
+  - apiUrl: https://evm-explorer.alephzero.org/api
+    family: blockscout
+    name: Aleph Zero Explorer
+    url: https://evm-explorer.alephzero.org
+blocks:
+  confirmations: 1
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 787
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Aleph Zero
+domainId: 787
+gasCurrencyCoinGeckoId: aleph-zero
+name: alephzero
+nativeToken:
+  decimals: 18
+  name: AZERO
+  symbol: AZERO
+protocol: ethereum
+rpcUrls:
+  - http: https://rpc.alephzero.raas.gelato.cloud
```

Comment:
I'm seeing single RPC URLs for a lot of these. Any chance we can find other public ones?

Reply:
I tried finding some more for them in my first pass, was going to do another pass on rpc urls when the set of chains gets finalised